app/assets/javascripts/components/story/StoryDatePicker.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import DatePicker from 'react-datepicker';
+import moment from 'moment';
+
+export default class StoryDatePicker extends React.Component {
+  constructor (props) {
+    super(props)
+    moment.locale(I18n.locale);
```

Comment:
In the future we should move it somewhere else, maybe the app.js file, but it's ok here for now.